client/lib/purchases/index.js

```diff
@@ -101,6 +105,14 @@ function isRefundable( purchase ) {
 	return purchase.isRefundable;
 }
 
+function isRemovable( purchase ) {
+	if (  isIncludedWithPlan( purchase ) ) {
+		return false;
+	}
+
+	return ( isExpired( purchase ) && purchase.isCancelable );
```

Comment:
Why are we accessing the `isCancelable` property here rather than using the `isCancelable` function? I think that creates some confusion. I'm confused anyway!